config/packages/pwa.yaml

```diff
@@ -33,15 +33,16 @@ pwa:
         src: "sw.js"
         skip_waiting: true
         workbox:
-            warm_cache_urls:
-                - {path: "app_planet_show", params: {id: 1}}
-                - {path: "app_planet_show", params: {id: 2}}
-                - {path: "app_planet_show", params: {id: 3}}
-                - {path: "app_planet_show", params: {id: 4}}
-                - {path: "app_planet_show", params: {id: 5}}
-                - 'app_homepage'
-                - 'app_voyage_index'
-                - 'app_planet_index'
-#                - 'app_about'
-            page_fallback: 'app_homepage'
-            static_regex: '/\.(css|js|json|xml|txt|mp3|map|webmanifest)$/'
```

Comment:
You should let the default behavior. It also includes images served by asset mapper.

Reply:
I remember adding that when we were discussing caching mp3s.  Adding pages to the page cache from a service feels like the next most important thing now.  I think there's an issue open already.